ft-logic/src/lib.rs

```diff
@@ -75,6 +75,35 @@ impl FTLogic {
                         self.approve(transaction_hash, account, &approved_account, amount)
                             .await;
                     }
+                    Action::PermitUnsigned {
+                        owner_account: _,
+                        approved_account: _,
+                        amount: _,
+                    } => {
+                        // TODO(m.strizhak) Where to report it?
+                        reply_err();
+                    }
+                    Action::Permit {
```

Comment:
There is a problem with that signature. Anyone can send it as many times as they want after it has appeared on the chain. One of the possible solution is to add the time during which the signature is valid.

Reply:
Solved. Protected with permit id